lib/dsp_blueprint_parser.rb

```diff
@@ -15,6 +15,8 @@
 require_relative 'dsp_blueprint_parser/parser'
 require_relative 'dsp_blueprint_parser/data_sections'
 
+BLUEPRINT_TYPE = /(BLUEPRINT|DYBP):/
```

Comment:
Are Ruby Regex object immutable? I wonder if this needs a freeze. Not on my w/ Ruby on it to check.